metriq/client.py

```diff
@@ -442,3 +444,17 @@ def method_names_get(self) -> List[Method]:
             Method(**r)
             for r in response["data"]
         ]
+
+    @handler
+    def platform_get(self) -> List[Environment]:
+        """Return a List of Environment objects.
+
+        Returns:
+            List: List of Environment objects.
+        """
+        response = self.http.get(f"/environment/")
```

Comment:
Even though the keyword is reserved, the back end route is still `/platform/`, for now, not `/environment/`.

Reply:
Done